### ISSUE TITLE
Update / expand / remove indicator data

### ISSUE THREAD

**Issue by davidheyman** [axismaps/ridership#71]
### ACS
- [x] Total Population
- [x] Population Density
- [x] Percent Foreign Born
- [x] Percent White
- [x] Percent Black
- [x] Percent Asian
- [x] Percent Hispanic/Latino
- [x] Percent ages 75+
- [x] Median Household Income
- [ ] **NEW:** Average monthly housing rent
- [x] Percent Households with No Vehicle
- [x] Percent Commute by Driving Alone
- [x] Percent Commute by Public Transit
- [x] **REMOVE:** Percent Commute by Carpooling

### LEHD
- [x] Jobs Density (update all data from 2010)
- [x] Job and Population Combined Density

### NTD
- [x] Total Ridership (Unlinked Passenger Trips)
- [x] Bus Ridership
- [x] Rail Ridership
- [x] **NEW:** Paratransit Ridership
- [x] Vehicle Revenue Miles (all modes, bus, rail)
- [x] Expenses (all modes, bus, rail)
- [x] Average Speed (all modes, bus, rail)
- [x] Farebox Recovery (all modes, bus, rail)
- [x] Average Fare (all modes, bus, rail)
- [x] Riders per Vehicle Revenue Miles (trips/mi) (all modes, bus, rail)
- [x] Minimum headways
- [x] Average Trip Length

### EIA
- [ ] Statewide Gas Prices

**Comment by awoodruff** [comment]
@davidheyman When the demographic data updates happen, can you double check that zero and `null` are correctly distinguished? In the current data table, there is at least a little bit of confusion between the two, though possibly just in the LEHD ones (e.g. 2016 data is missing but is all represented as 0)

**Comment by davidheyman** [comment]
@awoodruff I'm not sure how to get the census data updated. We need to add rent and remove commute by carpooling from the menu.

**Comment by awoodruff** [comment]
@davidheyman for some reason the dropdown list is hard-coded in here: https://github.com/axismaps/ridership/blob/master/src/js/data/dataDropdownItems.js. (I think it would be no big deal to load it from an equivalent external JSON if that makes things easier.)

I've made that change but the rent indicator may need additional text for its `meta` property which shows up on mouseovers. For now I just copied the "Average monthly housing rent" name into there.
